config/rbac/byoh_csr_creator_clusterrolebinding.yaml

```diff
@@ -10,3 +10,6 @@ subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: Group
   name: system:bootstrappers:byoh
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: byoh:hosts
```

Comment:
Is this YAML generated through templates? 
if yes, it will be good to only add this permission when `CERTIFICATE_ROTATION` is set/enabled.
If not, this looks good 👍

Reply:
I don't think its generated yaml.

cc @dharmjit to keep me honest here.

Reply:
Only the RBAC manifests for `BYOH` CRDs are managed via `kubebuilder` markers. These additional RBACs are added/updated manually.